ZeeHomeScreen/Classes/Models/ComponentModel.swift

```diff
@@ -39,7 +39,7 @@ import ApplicasterSDK
     @objc open var containerType: String?
     //, ComponentModelProtocol {
     
-    open var parentModel: ComponentModel?
+    open weak var parentModel: ComponentModel?
```

Comment:
Retain cycles in ComponentModels, CellModels.

I have never seen such a thing :D

<img width="940" alt="memory_leak" src="https://user-images.githubusercontent.com/849632/89639001-14e95800-d8ad-11ea-8fbb-13154598aff8.png">